### DISCUSSION TITLE
Could I close the longer breath tube?

### DISCUSSION THREAD

**Discussion by Ekolide** [cardonabits/haxo-hw#81] [Q&A]
Hi!

One of the plastic tubes on the Haxophones directs air to the pressure sensor. That's an important task. The other one directs air away from the instrument.

If I remove the long tube and plug the hole with something, and start leaking air through my mouth instead, would there be any impact on the instrument? I mean, is there any very specific purpose to leading air the way it is now?

Thanks!

**Comment by jcard0na** [comment]
Hi @Ekolide,

The venting tube serves two purposes: (1) let the air flow with some resistance, to resemble the feeling of a real saxophone and (2) provide an escape path for the moisture, away from the pressure sensor.  If you don't care about (1), maybe because you are looking for a different breath control experience, you should make sure that you don't lose (2).  If the short tube attached to the sensor gets clogged with liquid, I don't think it will work correctly.  That said, this is all hypothetical:  I have not tried it myself, and I'd love to know how things turn out if you try it.  The worst that could happen is that you damage the pressure sensor, which is relatively easy to replace.

Best,
Javier
